### ISSUE TITLE
[Crash] Crash when click Upload local File button 

### ISSUE THREAD

**Issue by trihung93** [filestack/filepicker-android#106]
Project : io.filepicker.sample

SamSung tablet 
Model SM-T561Y
Android ver : 4.4.4
I think it should be check url or return on onFileUploadError 

```
10-27 12:27:24.709 21578-21578/io.filepicker.sample I/Timeline: Timeline: Activity_idle id: android.os.BinderProxy@41d758f8 time:163594668
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/System.err: java.io.FileNotFoundException: No content provider: PUT PATH TO LOCAL FILE HERE - something like content://...
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/System.err:     at android.content.ContentResolver.openTypedAssetFileDescriptor(ContentResolver.java:1049)
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/System.err:     at android.content.ContentResolver.openAssetFileDescriptor(ContentResolver.java:904)
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/System.err:     at android.content.ContentResolver.openInputStream(ContentResolver.java:629)
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/System.err:     at io.filepicker.utils.FilesUtils.cacheRemoteFile(FilesUtils.java:174)
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/System.err:     at io.filepicker.utils.FilesUtils.getPath(FilesUtils.java:68)
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/System.err:     at io.filepicker.services.ContentService.handleActionUploadFile(ContentService.java:225)
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/System.err:     at io.filepicker.services.ContentService.onHandleIntent(ContentService.java:142)
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/System.err:     at android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:65)
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/System.err:     at android.os.Handler.dispatchMessage(Handler.java:102)
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/System.err:     at android.os.Looper.loop(Looper.java:136)
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/System.err:     at android.os.HandlerThread.run(HandlerThread.java:61)
10-27 12:27:25.650 21578-21600/io.filepicker.sample W/dalvikvm: threadid=12: thread exiting with uncaught exception (group=0x4178fbc0)
10-27 12:27:25.650 21578-21600/io.filepicker.sample E/AndroidRuntime: FATAL EXCEPTION: IntentService[ContentService]
                                                                      Process: io.filepicker.sample, PID: 21578
                                                                      java.lang.NullPointerException
                                                                          at java.io.File.fixSlashes(File.java:185)
                                                                          at java.io.File.<init>(File.java:134)
                                                                          at io.filepicker.services.ContentService.waitForFile(ContentService.java:259)
                                                                          at io.filepicker.services.ContentService.handleActionUploadFile(ContentService.java:226)
                                                                          at io.filepicker.services.ContentService.onHandleIntent(ContentService.java:142)
                                                                          at android.app.IntentService$ServiceHandler.handleMessage(IntentService.java:65)
                                                                          at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                          at android.os.Looper.loop(Looper.java:136)
```

**Comment by janhudzik** [comment]
Fixed in 5e76ac432b7f135b27a21ed6937cdebd31942247. Callback onFileUploadError() will be invoked if no local file is found under the specified path.

**Comment by trihung93** [comment]
tks .
